app/assets/javascripts/components/tabs.js

```diff
@@ -0,0 +1,106 @@
+/*
+    We used the thorough analysis and scripting of Jason Kiss for these ARIA tabs.
+    See Accessible ARIA Tabs: http://www.accessibleculture.org/articles/2010/08/aria-tabs/
+    for his analysis, examples and conclusions. Thanks, Jason!
+*/
+$(function() {
```

Comment:
This syntax will execute this code on `document.ready()` which we shouldn't have to do. Following the pattern in `app/assets/javascripts/components/carousel.js` should be okay:

```js
(function(window, $) {
  'use strict';

  // your component's code here…
})(window, jQuery);
```

This pattern is known as an [IIFE](http://benalman.com/news/2010/11/immediately-invoked-function-expression/). Passing in `window` and `jQuery` (as `$`) is a small convenience to [Uglifier](https://github.com/lautis/uglifier) which will treat `window` and `$` within the context of this function as local variables (rather than the global objects they represent) and more effectively munge them.

(Yeah, `jQuery` being `$` in our function can't get much shorter, but… 🤷‍♂️ )